bitherj/src/main/java/net/bither/bitherj/PrimerjSettings.java

```diff
@@ -74,6 +80,11 @@
     public static final int btfP2shHeader = 40;
     public static final int btpP2shHeader = 58;
 
+    public static int getAddressHeader(boolean isTestNet) {
+        if(isTestNet) return testAddressHeader;
```

Comment:
i see you always call utils for the isTestNet, maybe better to call utils here instead so caller doesn't need to

Reply:
Good suggestion, I have modified this